starlite/middleware/utils.py

```diff
@@ -51,6 +51,11 @@ def should_bypass_middleware(
         return True
     if exclude_opt_key and scope["route_handler"].opt.get(exclude_opt_key):
         return True
-    if exclude_path_pattern and exclude_path_pattern.findall(scope["path"]):
+    path = (
+        scope["path"]
+        if scope.get("mount_path") is None
```

Comment:
I made this check instead of `scope["mount_path"]` to make sure that a middleware would work even if it is used outside of Starlite with its `scope`.